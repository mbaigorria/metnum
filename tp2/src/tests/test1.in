0 0.85 0 ../tests/web-Stanford.txt 0.00001
