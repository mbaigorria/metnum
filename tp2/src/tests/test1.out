0.368151
0.141810
0.287961
0.202078
